Packs/CiscoUmbrellaReporting/Integrations/CiscoUmbrellaReporting/CiscoUmbrellaReporting_test.py

```diff
@@ -0,0 +1,802 @@
+import pytest
+import json
+import io
+from CommonServerPython import DemistoException
+from CiscoUmbrellaReporting import Client, get_destinations_list_command, \
+    get_categories_list_command, get_identities_list_command, get_file_list_command, \
+    get_threat_list_command, \
+    get_event_types_list_command, get_activity_list_command, \
+    get_activity_by_traffic_type_command,\
+    get_summary_list_command, pagination
+
+client = Client(
+    base_url="http://test.com",
+    organisation_id="1234567",
+    secret_key="test_123",
+    client_key="test@12345",
+    proxy=False
+)
+
+
+def util_load_json(path):
+    with io.open(path, mode='r') as f:
+        return json.loads(f.read())
+
+
+DESTINATION_LIST_RESPONSE = util_load_json('test_data/destination_data.json')
+CATEGORY_LIST_RESPONSE = util_load_json('test_data/category_data.json')
+IDENTITY_LIST_RESPONSE = util_load_json('test_data/identity_data.json')
+FILE_LIST_RESPONSE = util_load_json("test_data/file_data.json")
+THREAT_LIST_RESPONSE = util_load_json("test_data/threat_data.json")
+EVENT_TYPE_LIST_RESPONSE = util_load_json("test_data/event_type_data.json")
+ACTIVITY_LIST_RESPONSE = util_load_json("test_data/activity_data.json")
+ACTIVITY_DNS_LIST_RESPONSE = util_load_json("test_data/activity_dns_data.json")
+SUMMARY_LIST_RESPONSE = util_load_json("test_data/summary_data.json")
+DESTINATION_SUMMARY_LIST_RESPONSE = util_load_json(
+    "test_data/destination_summary_data.json")
+ACCESS_TOKEN_RESPONSE = util_load_json("test_data/access_token_data.json")
+ACTIVITY_FIREWALL_LIST_RESPONSE = util_load_json("test_data/file_data.json")
+PAGINATION_DATA_LIST = util_load_json("test_data/pagination_data.json")
+PAGE_1_PAGE_SIZE_5_LIST = util_load_json("test_data/page_1_paze_size_5.json")
+PAGE_2_PAGE_SIZE_5_LIST = util_load_json("test_data/page_2_paze_size_5.json")
+
+
+@pytest.mark.parametrize('raw_response, expected', [(DESTINATION_LIST_RESPONSE,
+                                                    DESTINATION_LIST_RESPONSE)])
+def test_get_destinations_list_command(mocker, raw_response, expected):
+    """
+    Tests get_destinations_list_command function.
+
+        Given:
+            - mocker object.
+            - raw_response test data.
+            - expected output.
+
+        When:
+            - Running the 'get_destinations_list_command'.
+
+        Then:
+            -  Checks the output of the command function with the expected output.
+    """
+    mocker.patch.object(client, 'query', side_effect=[raw_response] * 5)
+    args = {
+        "limit": 5,
+        "from": 1662015255000,
+        "to": 1662447255000,
+        "offset": 0
+    }
+
+    command_results = get_destinations_list_command(client, args)
+    # results is CommandResults list
+    context_detail = command_results.to_context()['Contents']
+    assert context_detail == expected.get("data")
```

Comment:
Please add asserts also for the readable output of the command. 
Apply to all commands tests.

Reply:
@ShacharKidor Added